typescript/web/src/components/website/Movie/movie.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Heading, Flex, Center, AspectRatio } from "@chakra-ui/react";
+import * as React from "react";
+
+export const Movie = () => {
+  return (
+    <Box id="Movie" as="section">
```

Comment:
Why do you need to set an `id`?
If really needed, can you please make-it lower-case instead?

Reply:
done